Doc/library/string.rst

```diff
@@ -65,6 +65,10 @@ The constants defined in this module are:
    combination of :const:`digits`, :const:`ascii_letters`, :const:`punctuation`,
    and :const:`whitespace`.
 
+.. data:: visible
+
+   String of non-blank ASCII characters. This is a combination of
+   :const:`digits`, :const:`ascii_letters`, and :const:`punctuation`.
```

Comment:
```suggestion
   :const:`digits`, :const:`ascii_letters`, and :const:`punctuation`.
   
   .. versionadded:: 3.14
```

Reply:
Ah sorry, for this, there are some invisible whitespaces either at the end of my suggestions or on blank lines (hence the test failures).